draft-ietf-quic-transport.md

```diff
@@ -3056,9 +3056,9 @@ An endpoint MUST NOT send a stateless reset that is three times or more larger
 than the packet it receives to avoid being used for amplification.
 {{reset-looping}} describes additional limits on stateless reset size.
 
-Endpoints MUST discard packets that are too small to be valid QUIC packets.
-With the set of AEAD functions defined in {{QUIC-TLS}}, packets that are smaller
-than 21 bytes are never valid.
+Endpoints MUST discard packets that are too small to be valid QUIC packets.  To
+give an example, with the set of AEAD functions defined in {{QUIC-TLS}}, short
+header packets that are smaller than 21 bytes are never valid.
```

Comment:
I like this new text, but I think we need to be explicit about what the minimum length is for long headers (including VN).

Reply:
I'm not sure if we can provide exact limits, because the minimum valid size of packets depend on:
* packet type
* CID being used by the endpoint
* extensions being negotiated

In practice, I'd assume that endpoints do not hard-code the numbers provided here (e.g., 21), but rather try to parse the structure of the packets, and discard if they fail. Assuming that's the way people implement, it is not important to explain what the numbers are.